test/TritonGPU/rewrite-tensor-pointer-tma.mlir

```diff
@@ -0,0 +1,96 @@
+// RUN: ENABLE_TMA=1 triton-opt %s -split-input-file -tritongpu-rewrite-tensor-pointer=compute-capability=90 | FileCheck %s
```

Comment:
This test seems way bigger than we need. And it does use `max_divisibility` ?

Reply:
Okay!